src/data/full-quiz.ts

```diff
@@ -1,4 +1,26 @@
 const fullQuiz = [
+  {
+    Question:
+      "What is the basic language of a computer?",
+    Answer: "Binary",
+    Distractor1: "HyperText Markup Language",
+    Distractor2: "Cascading Style Sheets",
+    Distractor3: "JavaScript",
+    Explanation:
+      "Computers are simplistic and understand binary, other languages (such as Python, Java, etc) were created to help humans write instructions for computers and interpret it.",
```

Comment:
I think the explanation should be two sentences. 

Proposed solution: 
Computers are simplistic and understand binary. Programming languages such as Python and Java were created to help humans write instructions for computers and interpret it.